website/docs/r/app_service_slot_custom_hostname_binding.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_slot_custom_hostname_binding"
+description: |-
+  Manages a Hostname Binding within an App Service Slot.
+
+---
+
+# azurerm_app_service_slot_custom_hostname_binding
+
+Manages a Hostname Binding within an App Service Slot.
+
+## Example Usage
+
+```hcl
+resource "random_id" "server" {
+  keepers = {
+    azi_id = 1
+  }
+
+  byte_length = 8
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "some-resource-group"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_plan" "example" {
+  name                = "some-app-service-plan"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  sku {
+    tier = "Standard"
+    size = "S1"
+  }
+}
+
+resource "azurerm_app_service" "example" {
+  name                = random_id.server.hex
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+}
+
+resource "azurerm_app_service_slot" "example" {
+  name                = "staging"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_name    = azurerm_app_service.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+}
+
+resource "azurerm_app_service_slot_custom_hostname_binding" "example" {
+  app_service_slot_id = azurerm_app_service_slot.example.id
+  hostname            = "www.mywebsite.com"
+}
+```
```

Comment:
these example usages are intended to be examples of how to use them than copy/pastable examples, so can we make this:

```suggestion
```hcl
resource "azurerm_resource_group" "example" {
  name     = "some-resource-group"
  location = "West Europe"
}

resource "azurerm_app_service_plan" "example" {
  name                = "some-app-service-plan"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name

  sku {
    tier = "Standard"
    size = "S1"
  }
}

resource "azurerm_app_service" "example" {
  name                = "some-app-service"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  app_service_plan_id = azurerm_app_service_plan.example.id
}

resource "azurerm_app_service_slot" "example" {
  name                = "staging"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  app_service_name    = azurerm_app_service.example.name
  app_service_plan_id = azurerm_app_service_plan.example.id
}

resource "azurerm_app_service_slot_custom_hostname_binding" "example" {
  app_service_slot_id = azurerm_app_service_slot.example.id
  hostname            = "www.mywebsite.com"
}
```
```

Reply:
fixed